keybot/platform_linux.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package main
+
+import (
+	"github.com/keybase/slackbot"
+	"log"
+)
+
+func setEnvCommand(name string, val string) slackbot.ExecCommand {
+	log.Printf("WARNING: setEnvCommand(%q, %q) is a NO-OP on Linux", name, val)
+	return slackbot.NewExecCommand("true", []string{}, false, "Set the env")
+}
+
+func buildStartCommand() slackbot.ExecCommand {
+	return slackbot.NewExecCommand("bash", []string{"-c", "echo not implemented; false"}, false, "Perform a build")
```

Comment:
There's some duplication in the description strings here. We could refactor things a bit more if you like @gabriel. I could go either way.